test/units/plugins/action/test_action.py

```diff
@@ -627,6 +627,6 @@ def test_action_base_sudo_only_if_user_differs(self):
         try:
             play_context.remote_user = 'root'
             action_base._low_level_execute_command('ECHO SAME', sudoable=True)
-            play_context.make_become_cmd.assert_called_once_with("ECHO SAME", executable='/bin/sh')
+            play_context.make_become_cmd.assert_called_once_with("ECHO SAME", executable=None)
```

Comment:
the reason i left as `None` in the test is that it is called with `None` but does not result in it (diff test needed for that)